src/main/java/org/springframework/integration/dsl/LambdaMessageProcessor.java

```diff
@@ -125,6 +126,9 @@ public Object processMessage(Message<?> message) {
 		try {
 			return this.method.invoke(this.target, args);
 		}
+		catch(InvocationTargetException e) {
+			throw new MessageHandlingException(message, e.getCause());
+		}
```

Comment:
Right. At all in SI, when we get deal with `MessageHandlingException` and this `InvocationTargetException` would be overhead